datasync/chaindatafetcher/kafka/kafka_test.go

```diff
@@ -0,0 +1,123 @@
+// Copyright 2020 The klaytn Authors
+// This file is part of the klaytn library.
+//
+// The klaytn library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The klaytn library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the klaytn library. If not, see <http://www.gnu.org/licenses/>.
+
+package kafka
+
+import (
+	"encoding/json"
+	"math/rand"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/Shopify/sarama"
+	"github.com/stretchr/testify/suite"
+)
+
+type KafkaSuite struct {
+	suite.Suite
+	kfk      *Kafka
+	consumer sarama.Consumer
+	topic    string
+}
+
+func (s *KafkaSuite) SetupTest() {
+	config := DefaultKafkaConfig
+	config.brokers = []string{"kafka:9094"}
+	kfk, err := NewKafka(config)
+	s.NoError(err)
+	s.kfk = kfk
+
+	consumerConfig := sarama.NewConfig()
+	consumer, err := sarama.NewConsumer(config.brokers, consumerConfig)
+	s.NoError(err)
+	s.consumer = consumer
+	s.topic = "test-topic"
+}
+
+func (s *KafkaSuite) TestKafka_CreateAndDeleteTopic() {
+	// no topic to be deleted
+	err := s.kfk.admin.DeleteTopic(s.topic)
+	s.Error(err)
+	s.True(strings.Contains(err.Error(), sarama.ErrUnknownTopicOrPartition.Error()))
```

Comment:
Can't we do this like below?
```suggestion
	s.Equal(err, sarama.ErrUnknownTopicOrPartition)
```

Reply:
There is a wrapper function to make `TopicError`. I think the comparison of strings is enough.